lib/connection.cpp

```diff
@@ -467,6 +479,11 @@ void Connection::Private::completeSetup(const QString& mxId)
     emit q->stateChanged();
     emit q->connected();
     q->reloadCapabilities();
+#ifdef Quotient_E2EE_ENABLED
+    connectSingleShot(q, &Connection::syncDone, q, [=](){
```

Comment:
Something like `connectSingleShot(q, &Connection::syncDone, q, &Connection::Private::createDevicesList);` should probably work.

Reply:
AFAIU it does not, since it's part of Connection::Private which is not a QObject